migrations/versions/28a719de7ec9_.py

```diff
@@ -36,14 +36,14 @@ def upgrade():
         ),
         sa.PrimaryKeyConstraint("id"),
     )
-    op.drop_table("stakeholders")
+    op.drop_table("reviewers")
```

Comment:
please rebase against "main" branch, this change has already been incorporated by Sam